<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>742</width>
    <height>515</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="CodeEditor" name="plainTextEdit">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>701</width>
      <height>271</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="tabChangesFocus">
     <bool>false</bool>
    </property>
    <property name="undoRedoEnabled">
     <bool>true</bool>
    </property>
    <property name="readOnly">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="programOutput">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>310</y>
      <width>701</width>
      <height>101</height>
     </rect>
    </property>
    <property name="acceptDrops">
     <bool>false</bool>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(240, 240, 240);
border: 1px solid #777777;
color: #999999;</string>
    </property>
    <property name="undoRedoEnabled">
     <bool>false</bool>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
    <property name="plainText">
     <string/>
    </property>
    <property name="overwriteMode">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="processButton">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>420</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Process</string>
    </property>
   </widget>
   <widget class="QToolButton" name="clrOutput">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>320</y>
      <width>31</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>clr</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../../Downloads/clear.jpg</normaloff>../../Downloads/clear.jpg</iconset>
    </property>
   </widget>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>742</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionLexems_Table"/>
    <addaction name="actionConstants_Table"/>
    <addaction name="actionId_Table"/>
    <addaction name="actionLexical_Analyzer_Trace"/>
    <addaction name="actionPrecedence_Table"/>
   </widget>
   <widget class="QMenu" name="menuRuntime">
    <property name="title">
     <string>Runtime</string>
    </property>
    <addaction name="actionPolis"/>
    <addaction name="actionPolis_trace"/>
    <addaction name="actionPolis_execution"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuView"/>
   <addaction name="menuRuntime"/>
  </widget>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionLexems_Table">
   <property name="text">
    <string>Lexems Table</string>
   </property>
  </action>
  <action name="actionConstants_Table">
   <property name="text">
    <string>Constants Table</string>
   </property>
  </action>
  <action name="actionId_Table">
   <property name="text">
    <string>Id Table</string>
   </property>
  </action>
  <action name="actionLexical_Analyzer_Trace">
   <property name="text">
    <string>Synax Analyzer Trace</string>
   </property>
  </action>
  <action name="actionPrecedence_Table">
   <property name="text">
    <string>Precedence Table</string>
   </property>
  </action>
  <action name="actionPolis">
   <property name="text">
    <string>Polis</string>
   </property>
  </action>
  <action name="actionPolis_trace">
   <property name="text">
    <string>Polis trace</string>
   </property>
  </action>
  <action name="actionPolis_execution">
   <property name="text">
    <string>Polis execution</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>CodeEditor</class>
   <extends>QPlainTextEdit</extends>
   <header>codeeditor.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
